src/LiberisLabs.CompaniesHouse.IntegrationTests/Tests/CompanyFilingHistoryTests/CompanyFilingHistoryTestsInvalid.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
+
+namespace LiberisLabs.CompaniesHouse.IntegrationTests.Tests.CompanyFilingHistoryTests
+{
+    [TestFixture]
+    public class CompanyFilingHistoryTestsInvalid : CompanyFilingHistoryTestBase
+    {
+        private const string InvalidCompanyNumber = "ABC00000";
+
+        [SetUp]
+        protected override void When()
+        {
+            WhenRetrievingAnInvalidCompanyProfile();
+        }
+
+        [Test]
+        public void ThenTheProfileIsNotReturned()
+        {
+            Assert.That(_result.Data.Items, Is.Null);
```

Comment:
Hey - sure I can, but the actual data is not null. It is an envelope response with row counts, page number etc. If the company doesn't exist there is even a new status `filing-history-not-available-invalid-format` which could be used by the client apps.